src/main/resources/rules/civil/decision/decision.drl

```diff
@@ -105,8 +106,76 @@ rule "Disposable income under the limit "
 		
 end
 
+rule "Disposable income over the limit but DV waiver applies"
```

Comment:
This rule will no longer be necessary if the waiver limit is updated dynamically in the thresholds class